<launch>

  <!-- defines name of the namespace of the drone -->
  <arg name="UAV_NAME" default="$(optenv UAV_NAME uav1)" />

    <!-- Namespace - all topics, services and parameters will be remapped using this namespace as a prefix (eg.: "/waypoint_flier_simple/odom_uav_in" to "/uav1/waypoint_flier_simple/odom_uav_in") -->
  <group ns="$(arg UAV_NAME)">

    <node pkg="midas_ros" type="midas_node.py" name="midas" output="screen">

        <!-- ROS parameters config file -->
      <rosparam file="$(find midas_ros)/config/default.yaml" />

      <remap from="~image_compressed_in" to="camera_front_throttled/image_raw/compressed" />
      <remap from="~camera_info_in" to="camera_front_throttled/camera_info" />

      <remap from="~depth_image_out" to="~depth/image_raw" />
      <remap from="~camera_info_out" to="~depth/camera_info" />

    </node>

  </group>

</launch>
